include/avif/avif.h

```diff
@@ -113,7 +113,9 @@ typedef int avifBool;
 
 #define AVIF_SPEED_DEFAULT -1
 #define AVIF_SPEED_SLOWEST 0
-#define AVIF_SPEED_FASTEST 10
+
+// Libaom and Rav1e fastest speed setting is 10, while SVT-AV1 fastest profile is 13.
+#define AVIF_SPEED_FASTEST 13
```

Comment:
This may be a breaking change for some users who relied on this constant's value to never change. It should at least be described in CHANGES.md.

Reply:
Done